src/components/Tasks/QuickAdd.tsx

```diff
@@ -51,6 +58,7 @@ function QuickAdd(props: PropsType) {
         fullWidth
         value={title}
         onChange={handleChange}
+        InputLabelProps={{ className: useStyles().input }}
```

Comment:
Please move `useStyles()` to the top of the function. All hooks have to be called in every render and in the same order. Moving this to year increases the chances a feature change will mess this up.